2024.html

```diff
@@ -95,6 +95,7 @@ <h3>Kudos to these websites who <!-- @@ got --> plan to get naked in 2024!</h3>
             <li><a href="https://blakewatson.com/">Blake Watson</a></li>
             <li><a href="https://kayserifserif.place/">Katherine Yang</a></li>
             <li><a href="https://cssence.com/">Matthias Zöchling</a></li>
+            <li><a href="https://www.higby.io/">Branden Higby</a></li>
```

Comment:
Please move up (the list is sorted by last name).

Reply:
My apologies, I mistakenly appended it. It should be fixed now.